bindings/java/build-unix.xml

```diff
@@ -27,6 +27,42 @@
 		</java>
 	</target>
 
+	<condition property="online">
```

Comment:
This should probably go into build.xml and we should not do the check if someone specifies online=false as an ant argument.